pytorch_pfn_extras/handler/_logic.py

```diff
@@ -179,7 +162,14 @@ def __init__(
                     the gradient.
                 * ``'autocast'`` (bool):
                     If ``True``, ``torch.cuda.amp.autocast`` is enabled.
-                    Default is ``False``.
+                    Default is ``False``. This is deprecated in favor of
+                    ``autocast_options``.
+                * ``'autocast_options'`` (bool):
```

Comment:
How about accepting `autocast={'dtype': torch.bfloat16}` instead of adding a new option?

Reply:
Done!